docs/api/en/math/Plane.html

```diff
@@ -130,7 +130,8 @@ <h3>[method:Plane set]( [param:Vector3 normal], [param:Float constant] )</h3>
 			[page:Vector3 normal] - a unit length [page:Vector3] defining the normal of the plane.<br />
 			[page:Float constant] - the signed distance from the origin to the plane. Default is *0*.<br /><br />
 
-			 Sets the plane's [page:.normal normal] and [page:.constant constant] properties.
+			Sets the plane's [page:.normal normal] and [page:.constant constant] properties.<br>
+			Please note that this method only copies the values from the given normal.
```

Comment:
It would be less wordy to say it like this:

> Sets the plane's [page:.normal normal] and [page:.constant constant] properties by copying the values from the given normal.

Reply:
Thanks for your comments. I used your phrasing for this and for `Line3`.